python-sdk/src/astro/files/locations/azure/wasb.py

```diff
@@ -91,3 +93,21 @@ def openlineage_dataset_name(self) -> str:
         https://github.com/OpenLineage/OpenLineage/blob/main/spec/Naming.md
         """
         return urlparse(self.path).path
+
+    def get_new_path_for_snowflake_stage(self) -> str:
```

Comment:
Why not a property named `snowflake_stage_path`?

Reply:
Done.